### ISSUE TITLE
Stats parsing problem with latest jvb docker image

### ISSUE THREAD

**Issue by h3po** [systemli/prometheus-jitsi-meet-exporter#76]
The exporter is throwing http error 500 for a few days, I guess the colibri stats json format may have changed.

```
$ wget --content-on-error -qO- http://172.19.0.2:9888/metrics
json: cannot unmarshal number 0.0 into Go struct field videoBridgeStats.octo_send_bitrate of type int
```

```
$ docker image ls | grep -E "(jvb|exporter)"
jitsi/jvb                                 latest    137498120a93   8 days ago     417MB
systemli/prometheus-jitsi-meet-exporter   latest    d14fe3268686   4 months ago   6.28MB
```

```
$ wget -qO- http://172.19.0.6:8080/colibri/stats
{"inactive_endpoints":0,"inactive_conferences":0,"total_ice_succeeded_relayed":0,"colibri2":true,"total_loss_degraded_participant_seconds":3222,"bit_rate_download":8002.645,"local_active_endpoints":9,"muc_clients_connected":1,"total_participants":250,"total_packets_received":51042189,"rtt_aggregate":17.244370777777775,"packet_rate_upload":3843,"p2p_conferences":0,"total_aimd_bwe_expirations":11,"total_loss_limited_participant_seconds":2952,"preemptive_kfr_suppressed":3121,"local_endpoints":9,"octo_send_bitrate":0.0,"total_dominant_speaker_changes":6147,"endpoints_with_spurious_remb":5,"receive_only_endpoints":0,"total_colibri_web_socket_messages_received":324570,"octo_receive_bitrate":0.0,"version":"2.1.617-ga8b39c3f","total_ice_succeeded":248,"total_colibri_web_socket_messages_sent":381266,"total_bytes_sent_octo":0,"total_data_channel_messages_received":0,"total_conference_seconds":125966,"num_eps_oversending":0,"bit_rate_upload":16168.803,"total_conferences_completed":64,"octo_conferences":0,"num_eps_no_msg_transport_after_delay":0,"endpoints_sending_video":6,"packet_rate_download":1586,"muc_clients_configured":1,"outgoing_loss":4.856443529274642E-5,"overall_loss":3.0744824108861274E-5,"conference_sizes":[0,0,0,1,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"total_packets_sent_octo":0,"total_layering_changes_received":137,"total_relays":0,"conferences_by_video_senders":[0,1,0,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"endpoints_with_high_outgoing_loss":0,"stress_level":0.1077,"jitter_aggregate":3.4615266111257337,"drain":false,"total_video_stream_milliseconds_received":347303572,"total_ice_succeeded_tcp":0,"octo_endpoints":0,"current_timestamp":"2022-02-11 09:10:56.988","total_packets_dropped_octo":0,"num_relays_no_msg_transport_after_delay":0,"conferences":2,"participants":9,"total_keyframes_received":68231,"average_participant_stress":0.01,"largest_conference":6,"total_packets_sent":71871333,"endpoints":9,"total_data_channel_messages_sent":0,"incoming_loss":0.0,"total_bytes_received_octo":0,"octo_send_packet_rate":0,"conferences_by_audio_senders":[0,0,0,1,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"total_conferences_created":66,"total_ice_failed":0,"preemptive_kfr_sent":1503,"threads":89,"videochannels":9,"total_packets_received_octo":0,"graceful_shutdown":false,"octo_receive_packet_rate":0,"total_bytes_received":27199708276,"total_loss_controlled_participant_seconds":185241,"total_partially_failed_conferences":0,"endpoints_sending_audio":9,"dtls_failed_endpoints":0,"total_bytes_sent":27328616916,"mucs_configured":1,"total_failed_conferences":0,"mucs_joined":1}
```

edit: initially I piped the json through `jq` to get linebreaks, but that also converted `"octo_send_bitrate":0.0` into an int

**Comment by bb-Ricardo** [comment]
having the same issue right now

**Comment by 0x46616c6b** [comment]
Can you check the newest release `1.1.8`?

**Comment by bb-Ricardo** [comment]
Hi, yes this looks much better again. Thank you for this quick fix.

Btw: I got a shirt of your avatar.

**Comment by h3po** [comment]
Thanks, fixed by #79.
